gkr/Cargo.toml

```diff
@@ -28,6 +28,7 @@ sha2.workspace = true
 halo2curves.workspace = true
 thiserror.workspace = true
 ethnum.workspace = true
+rand_chacha.workspace = true
```

Comment:
nits: this is a dev-dependency?

Reply:
unfortunately - this is not quite a dev-dependency - just to ensure that PCS (in benchmarking binary) are generated with same CRS, this one gets into the dependency

Reply:
oh i see. sounds good.